Wabbajack/View Models/Gallery/ModListGalleryVM.cs

```diff
@@ -24,12 +24,12 @@ public class ModListGalleryVM : BackNavigatingVM
 
         public ObservableCollectionExtended<ModListMetadataVM> ModLists { get; } = new ObservableCollectionExtended<ModListMetadataVM>();
 
+        private FiltersSettings settings;
```

Comment:
I think we can remove this member.  The filter settings are always available via walking the links to the settings object (`MWVM.Settings.FilterSettings`).  Having a second variable here means there's two places the same information is stored, which has a low (but unnecessary) potential of desync.  If we store the filter settings object here, but then later someone goes and creates a new filter settings object on the "official" settings object in MWVM.. well now the two systems are pointing/using different filter settings objects.

Reply:
If you wanted to have a member here for convenience, so that you didn't have to write `MWVM.Settings.FilterSettings` every time, then you can utilize C#'s properties system to make a "derivative" property that just "reroutes" to the same variable.  That way there's not actually a second variable that needs to be synced with.
```
private FiltersSettings settings => MWVM.Settings.FilterSetting;
```